homepage.html

```diff
@@ -0,0 +1,140 @@
+                <div class="badge-parent">
+                    <span class="material-icons"> shopping_cart </span>
+                    <div class="badge-with-icon">5</div>
+                </div>
+                
+                <div class="badge-parent">
+                    <span class="material-icons"> favorite_border </span>
+                    <div class="badge-with-icon">3</div>
+                </div>
+        
+                <div class="badge-parent">
+                    <span class="material-icons"> notifications_none </span>
+                    <div class="badge-with-icon">1</div>
+                </div>
+
+            </div>
+        </div>
+    </nav>
+    <main>
+        <section class="hero-img">
+            <img class="responsive-img obj-fit-cover" src="./assets/img/showHome5.jpg" alt="">
+            <div class="hero-overlay d-flex align-center">
+                <div class="hero-overlay-heading font-size-lg">
+                    <h4>Upto 25% off</h4>
+                    <h1 class="font-size-xlg">Neon <span class="font-size-lg text bg-gray-200 br-sm py-1 color-primary-600">One</span></h1>
+                    <button class="btn btn-outline bg-transperent font-size-regular">Shop Now</button>    
+                </div>
+            </div>
+        </section>
+        <section class="container" >
+               <h2 class="font-size-md mr-top-md" >Shop by categories</h2> 
+                <div class="categories-container">
+                    <div class="categories-media">
+                        <img class="responsive-img obj-fit-cover br-top-sm" src="/assets/img/man1.jpg" alt="no img">
+                        <a href="#" class="categories-media-overlay">
+                           <h2 class="color-white position-center">Men</h2>
+                            <!-- <button class="btn btn-outline bg-transperent font-size-regular color-white position-center">Shop Now</button>   -->
+                        </a>
+                    </div>
+                    <div class="categories-media">
+                        <img class="responsive-img obj-fit-cover br-top-sm" src="/assets/img/women1.jpg" alt="no img">
+                        <a href="#" class="categories-media-overlay">
+                            <h2 class="color-white position-center">Women</h4>
+                            <!-- <button class="btn btn-outline bg-transperent font-size-regular color-white position-center">Shop Now</button> -->
+                        </a>
+                    </div>
+                    <div class="categories-media">
+                        <img class="responsive-img obj-fit-cover br-top-sm" src="/assets/img/kid1.jpg" alt="no img">
+                        <a href="#" class="color-white categories-media-overlay">
+                            <h2 class="position-center">Teen</h2>
+                            <!-- <button class="btn btn-outline bg-transperent font-size-regular color-white position-center">Shop Now</button> -->
```

Comment:
remove commented code

Reply:
thanks for pointing out 👍
but i kept it for purpose